invenio_app_rdm/theme/templates/semantic-ui/invenio_requests/community-submission/index.html

```diff
@@ -0,0 +1,56 @@
+{# -*- coding: utf-8 -*-
+
+  This file is part of Invenio.
+  Copyright (C) 2016-2022 CERN.
+
+  Invenio is free software; you can redistribute it and/or modify it
+  under the terms of the MIT License; see LICENSE file for more details.
+#}
+
+{% extends "invenio_requests/details/index.html" %}
+
+{% block javascript %}
+  {{ super() }}
+{{ webpack['invenio-app-rdm-requests.js']}}
+{% endblock %}
+
+{% block request_topic %}
+  <h1 class="ui header">
+    <div class="ui label green">New submission</div>
+    {{ request.topic.metadata.title }}</h1><br/>
+  <div class="ui horizontal list">
+    {% for item in request.topic.metadata.creators %}
+      <div class="item">
+        <div class="content"><i class="user icon"></i>
+          {{ item.person_or_org.name }}</div>
+      </div>
+    {% endfor %}
+  </div>
+{% endblock request_topic %}
+
+{% block conversation %}
+  <div class="ui divider hidden"></div>
+  <div class="ui container">
+  <div class="ui top attached tabular menu" id="request-community-submission-tab">
+    <a class="active item" data-tab="conversation">
+      Conversation
+    </a>
+    <a class="item" data-tab="record">
+      Record
+    </a>
+  </div>
+
+  <div class="ui bottom attached tab segment active" data-tab="conversation">
```

Comment:
Aria attributes for tab panel example: 
https://github.com/inveniosoftware/invenio-app-rdm/blob/master/invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/details/details.html#L110